code/defines/procs/announce.dm

```diff
@@ -0,0 +1,121 @@
+/var/datum/announcement/priority/priority_announcement = new(do_log = 0)
+/var/datum/announcement/priority/command/command_announcement = new(do_log = 0, do_newscast = 1)
+
+/datum/announcement
+	var/title = "Attention"
+	var/announcer = ""
+	var/log = 0
+	var/sound
+	var/newscast = 0
+	var/channel_name = "Station Announcements"
+	var/announcement_type = "Announcement"
+	var/disable_newscasts = 1 // Bay also adds announcements to their newscaster system - set this to 0 to also use that system
```

Comment:
This is the same thing as line 9, just reversed. Should be enough to remove ````do_newscast = 1```` from announcement initializations (such as line 2 above).